src/components/gitOps/constants.ts

```diff
@@ -95,3 +96,10 @@ export const gitOpsRepoNotConfiguredWithEnforcedEnv = (env: string): string =>
     `Deployment to ‘${env}’ requires a GitOps repository. Please configure and try again.`
 export const gitOpsRepoNotConfigured =
     'GitOps repository is required to deploy using GitOps. You can deploy using helm or configure GitOps repository and try again.'
+
+export const PROVIDER__DOC_LINK_MAP = {
```

Comment:
```suggestion
export const PROVIDER__DOC_LINK_MAP: Record<GitProvider, string> = {
```

this will ensure in future any one adds new GitProvider, this will throw an error to add documentation